velox/functions/lib/MapFromEntries.cpp

```diff
@@ -94,14 +96,16 @@ class MapFromEntriesFunction : public exec::VectorFunction {
     auto& inputValueVector = inputArray->elements();
     exec::LocalDecodedVector decodedRowVector(context);
     decodedRowVector.get()->decode(*inputValueVector);
-    // If the input array(unknown) then all rows should have errors.
```

Comment:
I checked Spark's implementation and discovered that the two signatures below appear to be invalid in Spark. I wonder if we can remove the registration for these two signatures for Spark and no need to handle the difference with `throwOnNull_` flag here.

```
            // unknown -> map(unknown, unknown)
            exec::FunctionSignatureBuilder()
                .returnType("map(unknown, unknown)")
                .argumentType("unknown")
                .build(),
            // array(unknown) -> map(unknown, unknown)
            exec::FunctionSignatureBuilder()
                .returnType("map(unknown, unknown)")
                .argumentType("array(unknown)")
                .build(),
```